internal/web/templates/pages/package.html

```diff
@@ -73,8 +73,17 @@ <h2 class="text-reset">Package Details</h2>
               {{ .Manifest.LongDescription }}
             {{ end }}
           </span>
+          <br />
+          <span>
+            {{ if IsUpgradable .Package.Status.Version .LatestVersion }}
```

Comment:
As you already discovered, there is a nil pointer error here. You can fix it by changing the expression to `if or (eq .Package nil) (IsUpgradable .Package.Status.Version .LatestVersion)`

Reply:
Yup this solves the issue.